README.md

```diff
@@ -31,46 +30,43 @@ go get github.com/eleanorhealth/simpleforce
 
 ### Setup the Client
 
-A `client` instance is the main entrance to access Salesforce using simpleforce. Create a `client` instance with the
-`NewClient` function, with the proper endpoint URL:
+An `httpClient` instance is the main entrance to access Salesforce using simpleforce. Create an `httpClient` instance with the
```

Comment:
I don't know if `httpClient` in back ticks is referring to anything here? It might make sense to just remove this first sentence and change `httpClient` to `HTTPClient`.